### ISSUE TITLE
Searching the deleted files😤

### ISSUE THREAD

**Issue by Ash515** [Ash515/AshTech-AI_Personal_Voice_Assistant#11]
It is very little hard to search the deleted files so it would be nice if our AshTech would show the deleted file location. So try to add some functionalities like that.😁

**Comment by vaishnavi-1** [comment]
Do you mean that our AshTech should open Recycle Bin?

**Comment by Ash515** [comment]
Yes @vaishnavi-1 can you have any idea?

**Comment by vaishnavi-1** [comment]
I can edit the code and when the user says to show the deleted files then AshTech will show all the files in the Recycle bin. @Ash515 assign me. I will do this.

**Comment by Ash515** [comment]
OK @vaishnavi-1 do your best

**Comment by realhunter7869** [comment]
I would like to work on this issue

**Comment by realhunter7869** [comment]
Hey @Ash515 can you assign this to me

**Comment by vaishnavi-1** [comment]
@realhunter7869 I have done it already. I will create a PR by today itself. You can look at other issues.

**Comment by vaishnavi-1** [comment]
This is wrong @realhunter7869 . You were not assigned then also you did it and created PR. solve only those issues which are assigned to you.
@Ash515 Please don't merge the PR. Tell him this is wrong way. I was unable to do it before as I was ill, that doesn't mean @realhunter7869 you will do without informing.

**Comment by Ash515** [comment]
@realhunter7869 please do another issues man 😊

**Comment by realhunter7869** [comment]
Hey there @Ash515, I asked you for this issue 3 days before. You didn't replied to it so I thought maybe you would be busy so didn't assigned it to me. No offense but @vaishnavi-1, I don't think solving the issue is a wrong gesture. You may have also gone for enhancing the code further. Anyways, I don't have any issue if this pr is deleted. Maybe @vaishnavi-1 can add her code and I can enhance it a bit further. @Ash515 you can keep the other things of that commit and delete the recycle bin part. And please assign me this issue when @vaishnavi-1 is done with pr so that if possible I may enhance it furthur. Thanks 😊.

**Comment by vaishnavi-1** [comment]
Okay @realhunter7869 . After my PR will be merged, then you can make changes and enhance the code and then create a new PR.
Secondly, if someone is assigned any issue and you are not then you should not do that task because the person which has been assigned he will do it, you can create other issues and work on them and issues are assigned on first come first basis.

**Comment by realhunter7869** [comment]
Hey @vaishnavi-1, I also asked to work on this issue three days before itself. I didn't got a reply so thought that maybe @Ash515 didn't went through it and forgot to assign it to me. I have also been adding codes to previously solved issues which were not assigned to me just to make it more user friendly (because at the end the working of the project matters). I never got such comments from anyone. If you wanted to add that code you would have said it in a polite way. I don't think either @Ash515 or me would have any problem with it. This code was pending from a long time and also I already requested to assign it to me so I gave a solution to it. Well if I hurt you in any way, my apologies. I will contribute further code once its assigned and your pr is merged. 😊

**Comment by vaishnavi-1** [comment]
@realhunter7869  I was just telling you, when to solve the issue and when not. I am really sorry for being rude. I didn't had any intention to hurt you.

**Comment by realhunter7869** [comment]
No worries @vaishnavi-1. I will be contributing more when my exams are settled. Till then if you have any more upgradations do add on. And @Ash515, please assign this issue to me too. Have a great time ahead @Ash515 @vaishnavi-1 😊
